tests/lib/rules/no-empty.js

```diff
@@ -44,36 +44,173 @@ ruleTester.run("no-empty", rule, {
         { code: "try { foo(); } catch (ex) {} finally { bar(); }", options: [{ allowEmptyCatch: true }] }
     ],
     invalid: [
-        { code: "try {} catch (ex) {throw ex}", errors: [{ messageId: "unexpected", data: { type: "block" }, type: "BlockStatement" }] },
-        { code: "try { foo() } catch (ex) {}", errors: [{ messageId: "unexpected", data: { type: "block" }, type: "BlockStatement" }] },
-        { code: "try { foo() } catch (ex) {throw ex} finally {}", errors: [{ messageId: "unexpected", data: { type: "block" }, type: "BlockStatement" }] },
-        { code: "if (foo) {}", errors: [{ messageId: "unexpected", data: { type: "block" }, type: "BlockStatement" }] },
-        { code: "while (foo) {}", errors: [{ messageId: "unexpected", data: { type: "block" }, type: "BlockStatement" }] },
-        { code: "for (;foo;) {}", errors: [{ messageId: "unexpected", data: { type: "block" }, type: "BlockStatement" }] },
-        { code: "switch(foo) {}", errors: [{ messageId: "unexpected", data: { type: "switch" }, type: "SwitchStatement" }] },
+        {
+            code: "try {} catch (ex) {throw ex}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "block" },
+                type: "BlockStatement",
+                suggestions: [{
+                    messageId: "suggestComment",
```

Comment:
Since we are using `data` with these suggestion messages, we should also add `data` to suggestion objects in the test cases.

Reply:
Done.